stripe/src/main/java/com/stripe/android/model/StripeModel.java

```diff
@@ -18,4 +23,19 @@
 
     @Override
     public abstract boolean equals(@Nullable Object obj);
+
+    @NonNull
+    static List<String> jsonArrayToList(@Nullable JSONArray jsonArray) {
+        final List<String> list = new ArrayList<>();
+        if (jsonArray != null) {
+            for (int i = 0; i < jsonArray.length(); i++) {
+                try {
+                    list.add(jsonArray.getString(i));
+                } catch (JSONException ignored) {
```

Comment:
is this safe to do? should we change the type to `@Nullable` and return null if it's a malformed json? or is it guaranteed to be well-formed?

Reply:
Should be safe, well-formed, validated data from the server.